src/qml/controls/Header.qml

```diff
@@ -36,7 +36,7 @@ Control {
         Loader {
             Layout.fillWidth: true
             Layout.preferredWidth: 0
-            active: root.description.length > 0
+            active: root.description.length > 0 ?? false
```

Comment:
Is syntax correct?

> The nullish coalescing operator (`??`) is a logical operator that returns its right-hand side operand when its left-hand side operand is null or undefined, and otherwise returns its left-hand side operand.

I thought that `root.description.length` should be a left-hand side operand, no?

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence it is equivalent to `root.description.length > (0 ?? false)`.

Reply:
This is not right, you will get an error like "can't read length of undefined".

You want the optional chaining operator (https://github.com/tc39/proposal-optional-chaining) which is not available in Qt 5.15.2, see https://bugreports.qt.io/browse/QTBUG-77926.

Reply:
you are right, these was some sleepy programming :), hebasto's suggestion here is correct from my understanding: https://github.com/bitcoin-core/gui-qml/pull/125#discussion_r886584266